### DISCUSSION TITLE
Return global index data

### DISCUSSION THREAD

**Discussion by enricomosca96** [alvarobartt/investpy#307] [Q&A] [answered]
Hi everyone!
I am trying to use the feature that allows me to extract index data. 
When I try to download the data of these indexes: "MSCI Emerging Markets (MSCIEF)", I get this error: "ERR # 0034: country global indexes not found"
Can someone help me?
Below I leave the information of the index

Thanks so much
![Screenshot 2021-03-16 at 07 48 09](https://user-images.githubusercontent.com/80742374/111267245-fe1d9780-862b-11eb-8794-2fa10823bfcb.jpg)

**Comment by alvarobartt** [comment, marked as answer]
Hi @enricomosca96, in order to retrieve the information from that concrete index you need to introduce its name and the `country='world'`, as I'm tagging the country as `world` whenever Investing.com says `Global Indices`.

So that the function call should look like:

```python
import investpy

data = investpy.get_index_historical_data(index="MSCI Emerging Markets",
                                          country="world",
                                          from_date="01/01/2020",
                                          to_date="01/01/2021")

print(data.head())
```

For more information please check [this doc entry](https://investpy.readthedocs.io/_api/indices.html#investpy.indices.search_indices) which explains to you how to use the `investpy.search_indices()` function so that you can look up this information easily. But you should also check [this wiki page](https://github.com/alvarobartt/investpy/wiki/investpy.search_quotes()) to see how does the `investpy.search_quotes()` function works, as you can easily use the Investing.com search engine with it.

Hope this was useful to you! 🤗

**Reply by enricomosca96** [reply]
Thank you! :D
